pkg/network/protocols/postgres/model_linux.go

```diff
@@ -73,6 +73,21 @@ func (e *EventWrapper) Operation() Operation {
 	return e.operation
 }
 
+// extractParameters returns the string following the command
+func (e *EventWrapper) extractParameters() string {
```

Comment:
Perhaps a simple unit test could be added for this function?

Reply:
I've added unit test